install.sh

```diff
@@ -0,0 +1,152 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+cat <<EOF
+     _           _   _   _      
+ ___| |__  _   _| |_| |_| | ___ 
+/ __| '_ \\| | | | __| __| |/ _ \\
+\__ \\ | | | |_| | |_| |_| |  __/
+|___/_| |_|\\__,_|\\__|\\__|_|\\___|
+                                
+https://www.shuttle.rs
+https://github.com/shuttle-hq/shuttle
+
+Please file an issue if you encounter any problems!
+===================================================
+EOF
+
+if ! command -v curl &>/dev/null; then
+  echo "curl not installed. Please install curl."
+  exit
+fi
+
+REPO_URL="https://github.com/shuttle-hq/shuttle"
+LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' "$REPO_URL/releases/latest")
+LATEST_VERSION="${LATEST_RELEASE#*\"tag_name\":\"}"
+LATEST_VERSION="${LATEST_VERSION%%\"*}"
+
+_install_linux() {
+  echo "Detected Linux!"
+  echo "Checking distro..."
+  if (uname -a | grep -qi "Microsoft"); then
+    OS="ubuntuwsl"
+  elif ! command -v lsb_release &>/dev/null; then
+    echo "lsb_release could not be found. Falling back to /etc/os-release"
+    OS="$(grep -Po '(?<=^ID=).*$' /etc/os-release | tr '[:upper:]' '[:lower:]')" 2>/dev/null
+  else
+    OS=$(lsb_release -i | awk '{ print $3 }' | tr '[:upper:]' '[:lower:]')
+  fi
+  case "$OS" in
+  "arch" | "manjarolinux" | "endeavouros")
+    _install_arch_linux
+    ;;
+  "ubuntu" | "ubuntuwsl" | "debian" | "linuxmint" | "parrot" | "kali" | "elementary" | "pop")
+    # TODO: distribute .deb packages via `cargo-deb` and install them here
```

Comment:
Why? We just need to download one binary.

Reply:
Let's stick to what distributions use to keep things native.